icarusalg/gallery/MCTruthBase/CMakeLists.txt

```diff
@@ -1,19 +1,24 @@
+file(GLOB src_files *.cxx *.cpp)
```

Comment:
OK for first pass: CMake best practice is to avoid `GLOB`bed library source lists wherever possible.